recipes/allure-pytest/meta.yaml

```diff
@@ -0,0 +1,121 @@
+{% set name = "allure-python" %}
+{% set version = "2.13.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/allure-framework/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9db4db2133a52e48af043fbb3bb920a77886f64a6a1c8cd56adee41828487fc6
+
+build:
+  noarch: python
+  number: 0
+
+outputs:
+  - name: allure-python-commons
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - attrs >=16.0.0
+        - pluggy >=0.4.0
+    test:
+      imports:
+        - allure
+        - allure_commons
+  - name: allure-behave
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - behave >=1.2.5
+        - {{ pin_subpackage('allure-python-commons', exact=True) }}
+    test:
+      imports:
+        - allure_behave
+  - name: allure-nose2
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - nose2
+        - {{ pin_subpackage('allure-python-commons', exact=True) }}
+    test:
+      imports:
+        - allure_nose2
+  - name: allure-pytest
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - pytest >=4.5.0
+        - {{ pin_subpackage('allure-python-commons', exact=True) }}
+    test:
+      imports:
+        - allure_pytest
+  - name: allure-pytest-bdd
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - pytest >=4.5.0
+        - pytest-bdd >=3.0.0
+        - {{ pin_subpackage('allure-python-commons', exact=True) }}
+    test:
+      imports:
+        - allure_pytest_bdd
+  - name: allure-robotframework
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+        - setuptools-scm
+      run:
+        - python >=3.6
+        - robotframework
+        - {{ pin_subpackage('allure-python-commons', exact=True) }}
+    test:
+      imports:
+        - AllureLibrary
+        - allure_robotframework
+
+about:
+  home: https://github.com/allure-framework/allure-python
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Allure integrations for Python test frameworks
+
+extra:
+  recipe-maintainers:
+    - kenodegard
+    - nsoranzo
```

Comment:
@nsoranzo as the maintainer of https://github.com/conda-forge/allure-python-commons-feedstock are you still interested in being a maintainer here?

Reply:
allure-python-commons is a dependency of a bioconda package I co-maintain, so happy to be listed here, thanks!